src/dscanner/stats.d

```diff
@@ -52,3 +52,155 @@ ulong printLineCount(Tokens)(File output, string fileName, ref Tokens tokens)
 	output.writefln("%s:\t%d", fileName, count);
 	return count;
 }
+
+void printTokenDump(string fileName)
+{
+	import dmd.tokens;
+	import dmd.lexer;
+	import std.file;
+
+	auto input = readText(fileName);
```

Comment:
Why is this function reading the file again? It seems that we have already read the bytes here: https://github.com/Dlang-UPB/D-scanner/pull/13/files#diff-d6826361035218d94dfa90eb567fe01c5b904a35a8fdfe45278569b3617c3717R196

Reply:
The printfTokenDump function should receive a string a simply lex it. That would make the unittest a lot easier also.